sessionctx/variable/sysvar.go

```diff
@@ -1888,6 +1897,10 @@ var defaultSysVars = []*SysVar{
 			return nil
 		},
 	},
+	{Scope: ScopeGlobal | ScopeSession, Name: TiDBTxnAssertionLevel, Value: DefTiDBTxnAssertionLevel, PossibleValues: []string{AssertionOffStr, AssertionFastStr, AssertionStrictStr}, Hidden: true, Type: TypeEnum, SetSession: func(s *SessionVars, val string) error {
+		s.AssertionLevel = tidbOptAssertionLevel(val)
+		return nil
+	}},
```

Comment:
Does it need to be `Hidden: true` ?  We have accepted a proposal to [no longer use hidden variables](https://github.com/pingcap/tidb/blob/master/docs/design/2021-07-29-hidden-sysvars.md) since they cause issues for support.

The sysvar changes otherwise LGTM.